source/ios/AdaptiveCards/ADCIOSVisualizer/ADCIOSVisualizer/ViewController.m

```diff
@@ -56,6 +60,23 @@ - (IBAction)editText:(id)sender
                               @"V:|-40-[editView(==200)]-[buttonLayout]", nil];
     [ViewController applyConstraints:formats variables:viewMap];
 }
+- (IBAction)enableCustomRenderer:(id)sender
```

Comment:
>enableCustomRenderer [](start = 12, length = 20)

This should be called `toggleCustomRenderer`, since that's what it does.

Reply:
Ideally you would change "enable" to "toggle" everywhere. And in the UI, the button's title should switch from "Enable" to "Disable" when the custom renderer is enabled.


---
In reply to: [189055276](https://github.com/Microsoft/AdaptiveCards/pull/1445#discussion_r189055276) [](ancestors = 189055276)